docs/distribution/release-policy.md

```diff
@@ -0,0 +1,180 @@
+
+## Release Workflow
+Cutting a release for Enso proceeds as follows:
+
+1.  If no release branch exists for the current major version, one should be
+    created.
+2.  Release notes should be made up to date.
+3.  A commit representing the release should be tagged, and the tag pushed to
+    GitHub.
+4.  CI will create a draft release for this tag, as well as build and upload the
+    appropriate artefacts.
+5.  The release notes for the version being released should be copied into the
+    release body on GitHub.
+6.  The release must be verified by two members of the engine team, and the QA
+    team.
+7.  Once approval has been gained from these members, the release may be made
+    official.
+
+### Tag Naming
+Tags for releases are named as follows `enso-version`, where `version` is the
+semver string (see [versioning](#versioning)) representing the version being
+released.
+
+### GitHub Releases
+A release is considered _official_ once it has been made into a release on
+[GitHub](https://github.com/enso-org/enso/releases). Once official, a release
+may not be changed in any way, except to mark it as broken.
+
+#### Marking a Release as Broken
+We intend to _never_ delete a release from GitHub, as users may have projects
+that depend on specific versions of Enso. Instead, we provide a mechanism for
+marking releases as broken that works as follows:
+
+- An empty file named `broken` is uploaded to the release.
+- The release description is edited to visibly mark the release as broken.
+
+A broken release is one that _must not_ be downloaded by the launcher unless
+a project specifies _an exact version match_, and it _must not_ be used in new
+projects by the launcher unless _explicitly_ specified by the user as an exact
+version match.
+
+### Release Notes
+Release notes should contain a summary of the changes made between the last
+release and the current release. They should follow the template given below:
+
+```md
+# Enso x.y.z (YYYY-MM-DD)
+
+## Language
+
+- A list of language-level changes.
+
+## Type System
+
+- A list of type-system changes.
+
+## Interpreter
+
+- A list of changes to the Enso interpreter.
+
+## Runtime
+
+- A list of changes to the Enso runtime.
+
+## Tooling
+
+- A list of changes to the Enso language tooling.
+
+## Libraries
+
+- A list of changes to the Enso core libraries.
+
+## Stabilised Features
+
+- A list of stabilised APIs and/or features.
+
+## Misc
+
+- A list of miscellaneous changes.
+
+## Internal Only
+
+- A list of changes that do not have user-facing impact, but represent
+  significant improvements to the internals of Enso and related tools.
+
+```
+
+If there are no changes for a section, the section should contain a bullet point
+that reads "Nothing".
+
+The changelog file is an ongoing record of changes, and may diverge between
+`main` and the various release branches.
+
+## Version Support
+We aim to support a given major version for two years after the release of the
```

Comment:
I think this should be reduced (or omitted altogether) at the beginning. Supporting the first version for 2 years is crazy.